_envsetup.yml

```diff
@@ -1,8 +1,16 @@
 default:
   paths:
-    sdtm: '/sdtm/'
-    adam: '/submission/datasets/'
-    qc: '/adam/'
-    output: '/output/'
-    programs: '/submission/programs/'
-    functions: '/R/'
+    sdtm: !expr file.path(getwd(), "sdtm")
+    adam: !expr list(PROD = file.path(getwd(), "submission/datasets"), QC = file.path(getwd(), "adam"))
+    output: !expr file.path(getwd(), "output")
+    programs: !expr file.path(getwd(), "submission/programs")
+    functions: !expr file.path(getwd(), "R")
+    envsetup_environ: !expr Sys.setenv(ENVSETUP_ENVIRON = 'QC'); 'QC'
```

Comment:
There's no advantage of using `file.path(getwd(), "somepath")` versus just `./somepath` or `somepath` in an RStudio project.

Reply:
If that is the preference ... Updated.